### PR TITLE
Reduced the non-meaningful logs.

### PR DESCRIPTION
**Pull request by yhwen**
Fixes # .

### Description

Reduced the non-meaningful logs.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/private/fed/server/server_commands.py [hunk 1]
```diff
@@ -182,10 +182,11 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         shared_fl_ctx.set_public_props(copy.deepcopy(get_serializable_data(fl_ctx).get_all_public_props()))
         shareable.set_header(key=FLContextKey.PEER_CONTEXT, value=shared_fl_ctx)
 
-        self.logger.info(
-            f"return task to client.  client_name:{client.name}  task_name: {taskname}   task_id:{task_id}  "
-            f"sharable_header_task_id: {shareable.get_header(key=FLContextKey.TASK_ID)}"
-        )
+        if taskname != SpecialTaskName.TRY_AGAIN:
+            self.logger.info(
+                f"return task to client.  client_name:{client.name}  task_name: {taskname}   task_id:{task_id}  "
```
Comment (holgerroth) [inline nvflare/private/fed/server/server_commands.py:187]: add a space after `client_name:`

Comment (holgerroth) [reply nvflare/private/fed/server/server_commands.py:187]: and also `task_id:`

Comment (yhwen) [reply nvflare/private/fed/server/server_commands.py]: done

**Review by holgerroth — APPROVED** [review]
Looks good.

**Pull request merged**